Add PHP set-fingerprint code (#3038)
getsentry/sentry-docs: src/includes/set-fingerprint/basic/php.mdx

```php
try {
  // Run code that possibly throws an exception
} catch (\Throwable $e) {
  \Sentry\withScope(function (\Sentry\State\Scope $scope) use ($e) {

>     // Group errors together based on the message of the exception being thrown

Reviewer: ```suggestion
    // Group errors together based on a static string
```

>     // Group errors together based on the message of the exception being thrown

Reviewer: I still don't get this comment. There is no usage of the exception message whatsoever.

Author: 🤦 hope to have fixed it now correctly!

Reviewer: Perfextion
